drisl.src.html

```diff
@@ -70,12 +70,18 @@ <h2>CIDs in CBOR</h2>
       <h2>Debugging Considerations</h2>
       <p>
         It is often convenient to represent DRISL in textual formats for debugging
-        purposes, for example using JSON.
+        purposes.
       </p>
       <p>
-        One common convention for doing so is historically referred to as "DAG-JSON," which is a JSON expression
-        of the DRISL data in which CIDs are represented as the object
-        <code>{ "/": "string CID" }</code>, i.e., DASL CIDs in their base-32 ASCII representation ([[cid]]), NOT in their CBOR representation ([[cbor-tag42]])). Another option is
+        One common convention for debugging DRISL data in JSON is to use a JSON profile called DAG-JSON.
+        This profile can be translated to JSON using standard CBOR-JSON translation, but CBOR bytestrings will be represented as
+        <code>"/": { "bytes": String  * base64-encoded * }</code>
+        and tag-42 DRISL CIDs (whose values in CBOR are also bytestrings) will be represented as:
+        <code>"/": String * base58btc-encoded *</code>.
+        Note that legacy DAG-CBOR data MAY include a base32-encoded variant of older CIDs, which are not DRISL CIDs.
```

Comment:
```suggestion
        <code>"/": String * CID string format *</code>.
        Note that legacy DAG-JSON data MAY include a base58btc-encoded variant of older CIDs, which are not DRISL CIDs.
```

https://ipld.io/specs/codecs/dag-json/spec/

> CIDv1 is represented as a Multibase Base32 encoded string